packages/landing/src/index.html

```diff
@@ -9,8 +9,19 @@
     <title>Blueprint – A React UI toolkit for the web</title>
     <meta name="description" content="A React UI toolkit for the web">
     <meta name="viewport" content="width=device-width">
+
+    <!-- Facebook Sharing -->
+    <meta property="og:url" content="http://blueprintjs.com/" />
+    <meta property="og:type" content="website" />
+    <meta property="og:title" content="Blueprint - A React UI toolkit for the web" />
+    <meta property="og:description" content="Blueprint is a collection of elegant and easy-to-use React UI components from Palantir, freeing you to focus on building your product—-not the atomic pieces that comprise it." />
```

Comment:
Makes sense. Babel also has a fairly identical title and description for the card:
![image](https://cloud.githubusercontent.com/assets/443450/20992513/5300af00-bc99-11e6-91d8-d082d587c7f3.png)